test/asserts/big-number-assert_test.js

```diff
@@ -22,29 +22,77 @@ const Assert = BaseAssert.extend({
 describe('BigNumberAssert', () => {
   it('should throw an error if the input value is not a big number', () => {
     const choices = [[], {}, ''];
+    const options = [undefined, { validateSignificantDigits: true }, { validateSignificantDigits: false }];
 
-    choices.forEach(choice => {
+    options.forEach(option => {
+      choices.forEach(choice => {
+        try {
+          new Assert().BigNumber(option).validate(choice);
+
+          should.fail();
+        } catch (e) {
+          e.should.be.instanceOf(Violation);
+        }
+      });
+    });
+  });
+
+  it('should expose `assert` equal to `BigNumber`', () => {
+    const options = [undefined, { validateSignificantDigits: true }, { validateSignificantDigits: false }];
+
+    options.forEach(option => {
       try {
-        new Assert().BigNumber().validate(choice);
+        new Assert().BigNumber(option).validate();
 
         should.fail();
       } catch (e) {
-        e.should.be.instanceOf(Violation);
+        e.show().assert.should.equal('BigNumber');
       }
     });
   });
 
-  it('should expose `assert` equal to `BigNumber`', () => {
-    try {
-      new Assert().BigNumber().validate();
+  it('should accept a big number', () => {
+    const choices = [
+      1.01,
+      1.0111111111,
+      '1.0111111111'
+    ];
+    const options = [undefined, { validateSignificantDigits: true }, { validateSignificantDigits: false }];
 
-      should.fail();
-    } catch (e) {
-      e.show().assert.should.equal('BigNumber');
-    }
+    options.forEach(option => {
+      choices.forEach(choice => {
+        new Assert().BigNumber(option).validate(choice);
+      });
+    });
   });
 
-  it('should accept a big number', () => {
-    new Assert().BigNumber().validate(1.01);
+  it('should throw an error if a number has more than 15 significant digits and `validateSignificantDigits` is true or missing', () => {
```

Comment:
With my suggestion above, these 2 tests would have to have a conditional deciding which test to call, like:

```js
if (options === { validateSignificantDigits: false }) {
  it('should accept a big number with more than 15 significant digits', () => {
  });
} else {
  it('should throw an error if a number has more than 15 significant digits', () => {
  });
}
```